pmd-plsql/src/test/resources/net/sourceforge/pmd/lang/plsql/cpd/testdata/sample-plsql.txt

```diff
@@ -12,55 +12,55 @@ L3
     [IS]                                    1         2
```

Comment:
Please add for PLSQL an additional test case, similar like in the other languages (tabWidth.pls).

Reply:
I've added another test case!